### PR TITLE
replaced querySelector

### PR DESCRIPTION
**Pull request by DmytroDidukh**

### CODE REVIEW

File: client/src/components/comment-form/comment-form.js [hunk 1]
```diff
@@ -30,37 +32,28 @@ const CommentForm = ({
...
-      setTextareaFilled(false);
-    }
+    targetValue && setTextareaFilled(false);
+    setText(targetValue)
   }
 
   const logged = (
     <form className="form my-1 comments-form" onSubmit={addComment}>
       <h3> Leave a comment </h3>
+      <div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId}/></div>
```
Comment (VolodymyrTrach) [inline client/src/components/comment-form/comment-form.js]: **make it more readable:**
`<div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId}/></div>`

**change to** 
````
<div className='star'>
        <h6 className='rate'>
           Rate the product:
        </h6> 
        <StarsRating rating={rate} id={productId}/>
</div>
````

Comment (VolodymyrTrach) [inline client/src/components/comment-form/comment-form.js]: Rate the product: - hardcode. 
Make: 
const rateTitle = "Rate the product:"

**Review by VolodymyrTrach — CHANGES_REQUESTED** [review]

**Review by bandvov — APPROVED** [review]

**Review by oleksandrhodzevych — APPROVED** [review]

File: client/src/components/comment-form/comment-form.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import classNames from 'classnames'
+const CommentForm = ({productId, setComments, storeService, comments, rate}) => {
```
Comment (VolodymyrTrach) [inline client/src/components/comment-form/comment-form.js]: remove this line. It will not start!

**Comment by DmytroDidukh** [comment]
Done!

File: client/src/components/comment-form/comment-form.js [hunk 3]
```diff
@@ -30,37 +32,28 @@ const CommentForm = ({
...
+    targetValue && setTextareaFilled(false);
+    setText(targetValue)
   }
 
   const logged = (
     <form className="form my-1 comments-form" onSubmit={addComment}>
       <h3> Leave a comment </h3>
+      <div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId}/></div>
       <div className='star'>
         <h6 className='rate'> Rate the product: </h6>
```
Comment (VolodymyrTrach) [inline client/src/components/comment-form/comment-form.js]: it is the same as line 56. 
As a result, you will get Rate the product: ***** - two times.
remove one of them

**Review by VolodymyrTrach — CHANGES_REQUESTED** [review]

**Review by VolodymyrTrach — APPROVED** [review]

**Pull request merged**